test/mirage/describe/run.t

```diff
@@ -206,34 +148,20 @@ Describe before configure (no eval)
                 45 -> 35 [style="dashed"];
                 45 -> 33 [style="dashed"];
                 }
+<<<<<<< HEAD
+=======
+>>>>>>> f17e6c1e0 (Add Key.runtime to represent runtime keys)
+<<<<<<< HEAD
+=======
+>>>>>>> f17e6c1e0 (Add Key.runtime to represent runtime keys)
```

Comment:
these 6 lines above do not look correct.

Reply:
woops